src/modules/swap/ModuleSwap.vue

```diff
@@ -697,6 +739,12 @@ export default {
     buyEth() {
       window.open('https://ccswap.myetherwallet.com/#/', '_blank');
     },
+    switchTokens() {
```

Comment:
right now wen you click switch To field becomes empty. Is should have become the original From token

Reply:
oh never mind there is a different pr for that